pkg/scheduler/objects/application_state.go

```diff
@@ -205,6 +205,14 @@ func NewAppState() *fsm.FSM {
 					app.rejectedMessage = event.Args[1].(string) //nolint:errcheck
 				}
 			},
+			fmt.Sprintf("enter_%s", Running.String()): func(_ context.Context, event *fsm.Event) {
```

Comment:
This counts apps twice: Starting increased the value and now again in Running

Reply:
Actually it didn't count twice here, 'enter_Running' only increase app count if it's from Completing state. 

Accepting-> Starting (+1)
Starting -> Completing (-1)
Completing ->Running (+1)

But I agree that it's difficult to track. To simplify it, I will modify this PR to follow what you suggest in Jira.